lib/mirage.ml

```diff
@@ -1503,6 +1558,18 @@ end
 
 include Functoria_app.Make (Project)
 
+let in_parallel = function
+  | [singleton] -> singleton
+  | jobs ->
+    impl @@ object
+      inherit base_configurable
+      method ty = job
+      method name = "group"
+      method module_name = "In_parallel"
```

Comment:
It's preferable to give a module that at least exists, just in case. `Functoria_runtime` of `Pervasive`, for example.